compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -957,6 +971,15 @@ abstract class PrepJSInterop extends plugins.PluginComponent
       case ctor: MethodSymbol if ctor.isPrimaryConstructor => ctor
     }
 
+  /** Identify `ddef` accessor that have the shape described in issue #1899.
+   *  Note: This is an over approximation, read comment at call site.
+   */
+  private def isAnonymousJSObjectAccessorMethod(ddef: DefDef): Boolean = {
```

Comment:
I change it back to a tree while trying the `originalTree` scheme that did not work and then forgot to make it a symbol again.